services/state/schedule/schedule.proto

```diff
@@ -100,7 +100,7 @@ message Schedule {
      * The requested expiration time of the schedule as provided by the user.
      * The actual calculated expiration time may be "earlier" than this, but will not be later.
      */
-    Timestamp expiration_time_provided = 10;
+    Timestamp expiry_provided = 10;
```

Comment:
There is no good reason to shorten this, it's clearer _and more accurate_ spelled out (particularly as it *is* an expiration time, not an expiry).
Same applies to the calculated value.

Reply:
It is called expiry in other places. I was renaming for consistency. I could rename expiry -> expiration_time instead, if you think that's better.

Reply:
Actually it's mostly called expiry outside of services/state, where it can't be renamed.

Reply:
Per discussion, I will change _expiry_ to _expiration_seconds_